site/docs/latest/api/ledger-api.md

```diff
@@ -471,3 +471,304 @@ mvn exec:java -Dexec.mainClass=org.apache.bookkeeper.Dice
 Value = 3, isLeader = true
 Value = 1, isLeader = false
 ```
+
+## New API
```

Comment:
oh I forgot to make the new api document available in `latest` :(

Reply:
@sijie should I do something to address this comment ? Can you give me an hint please ?

Reply:
no. you don't need to do anything for this comment.